packages/builds/src/node/inversify.spec-config.ts

```diff
@@ -0,0 +1,9 @@
+/* Inversivy config file */
+import { Container /*, interfaces */ } from "inversify";
+import { IErrorParser, ErrorParser } from "./error-parser";
+
+const testContainer = new Container();
+
+testContainer.bind<ErrorParser>(IErrorParser).to(ErrorParser);
+
```

Comment:
And actually I think it should be called something else than builds... ? I know it wast the 1st target but now it's more about matching errors and I think that could go in core ?